peribolos/knative-sandbox.yaml

```diff
@@ -468,6 +474,7 @@ orgs:
           net-istio: write
           net-kourier: write
           sample-controller: write
+          async-component: write
```

Comment:
Please add it to the list for `admin` as well.

Reply:
@mattmoor done!